app/services/csv_helper.rb

```diff
@@ -0,0 +1,11 @@
+class CsvHelper
```

Comment:
```suggestion
class CSVHelper
```

Nit! I think the lowercase version looks weird.

Reply:
Yes I agree, I was having issues with Zeitwerk expecting to find it in `c_s_v_helper.rb` but worked out the fix for that now.